bower.json

```diff
@@ -1,10 +1,10 @@
 {
   "name": "purescript-matryoshka",
-  "homepage": "https://github.com/slamdata/purescript-matryoshka",
+  "homepage": "https://github.com/purescript-contrib/purescript-matryoshka",
   "license": "Apache-2.0",
   "repository": {
     "type": "git",
-    "url": "git://github.com/slamdata/purescript-matryoshka.git"
+    "url": "git://github.com/purescript-contrib/purescript-matryoshka.git"
```

Comment:
Oh! I keep forgetting about this, but for publishing future updates to this library the link here needs to be `https://`.

Reply:
Ah yes, good call!